openedx/core/djangoapps/catalog/management/commands/sync_course_runs.py

```diff
@@ -0,0 +1,55 @@
+"""
+Sync course runs from catalog service.
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from opaque_keys.edx.keys import CourseKey
+
+from catalog.utils import get_catalog_course_runs
+from content.course_overviews.models import CourseOverview
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_catalog_course_runs():
+    """
+    Fetch catalog course runs from catalog service for all course overview records.
+    """
+    course_overviews = CourseOverview.objects.all()
```

Comment:
@nasthagiri makes sense, I'm refactoring this to fetch all the `course_runs` from catalog. Thanks!